src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                + "age=" + age
+                + ", name='" + name
+                + '\''
+                + ", ownerName='" + ownerName
+                + '\''
+                + '}';
+    }
```

Comment:
```suggestion
  @Override
public String toString() {
  return "Cat{"
      + "age=" + age
      + ", name='" + name + '\''
      + ", ownerName='" + ownerName + '\''
      + '}';
}
```
Regarding string concatenation, I would say we can do that in a more readable way, not cluttering our code with unnecessary whitespaces and indentations.